doc/event_bus.md

```diff
@@ -42,10 +42,14 @@ every event subscriber will be fully loaded, even though it is not going to be u
 use SimpleBus\Message\CallableResolver\CallableCollection;
 use SimpleBus\Message\CallableResolver\ServiceLocatorAwareCallableResolver;
 
-// Provide a map of event names to callables. You can provide actual callables, or lazy-loading ones.
+// Provide a map of event names to callables. You can provide actual callables, or lazy-loading ones using a ServiceLocator
 $eventSubscribersByEventName = [
-    'Fully\Qualified\Class\Name\Of\Event' => [
-        ['event_subscriber_service_id', 'notify']
+    Fully\Qualified\Class\Name\Of\Event::class => [
+        $fqcn = My\Class\Name::class, //Will use ServiceLocator for instantiation and invoke $instance->notify()
```

Comment:
Well, that is how I got it configured now: 
- [the FQCN is a string, which will be instantiated by the serviceLocator](https://github.com/SimpleBus/MessageBus/blob/b866134c7c86022e021ebfa82bed46672aaa06d9/src/CallableResolver/ServiceLocatorAwareCallableResolver.php#L29)
- then the function is ran again, [the acquired object will be used and the notify method is executed](https://github.com/SimpleBus/MessageBus/blob/b866134c7c86022e021ebfa82bed46672aaa06d9/src/CallableResolver/ServiceLocatorAwareCallableResolver.php#L29) (ok, when it exists...)

What part is not true do you think?. Maybe the text should be 
> .. and invoke $instance->notify() if the notify() function exists